librecad/src/actions/rs_actionpolylinesegment.cpp

```diff
@@ -54,9 +54,9 @@ void RS_ActionPolylineSegment::init(int status) {
 	if (container->countSelected(true, entityType)) {
 		//find a selected entity
 		//TODO, find a better starting point
-		for (RS_Entity* e = container->firstEntity(RS2::ResolveAllButInserts);
+		for (RS_Entity* e = container->firstEntity(RS2::ResolveNone);
```

Comment:
@lordofbikes sorry, haven't seen your reply before writing mine - but looks like we're in agreement here.